ui/pages/build-status.js

```diff
@@ -265,13 +300,62 @@ const enhanceWithCoverageData = async reposByArea => {
   }, sumEnhanced);
 };
 
+const repoQuery = gql`
+  query getPipeline($project: String!) {
+    organization(login: $project) {
+      repositories(first: 100) {
+        nodes {
+          name
+          pullRequests(states: [OPEN], labels: ["dependencies"], first: 50) {
+            nodes {
+              createdAt
+              url
+            }
+            totalCount
+          }
+        }
+      }
+    }
+  }
+`;
+
+const getGithubOrganizationState = async () => {
+  console.log('GITHUB_TOKEN', JSON.stringify(process.env.GITHUB_TOKEN));
```

Comment:
Q: Do we really want this in the console?

Reply:
woooops! thank you very much for catching that... luckily so far the build is failing since it's not there... had to debug some env var issues locally 🙈

Reply:
Haha, I figured!